me.amankhanna.opendeck.yaml

```diff
@@ -0,0 +1,35 @@
+id: me.amankhanna.opendeck
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: opendeck
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: binary
```

Comment:
please use an actually relevant name. Like "opendeck".

Reply:
Will do. The Tauri docs use binary and I copy-pasted.

Reply:
the tauri documentation is wrong.

Reply:
Yes OK, I had fixed this already.